explorer/src/routes/transactions/transaction/+page.svelte

```diff
@@ -9,16 +9,19 @@
 
   const dataStore = createDataStore(duskAPI.getTransaction);
   const payloadStore = createDataStore(duskAPI.getTransactionDetails);
+  const marketStore = createDataStore(duskAPI.getMarketData);
 
   const getTransaction = () => {
     dataStore.getData($appStore.network, $page.url.searchParams.get("id"));
     payloadStore.getData($appStore.network, $page.url.searchParams.get("id"));
+    marketStore.getData($appStore.network);
```

Comment:
Does this change so often that we need to retrieve it for each `getTransaction` call?